eventkit_cloud/ui/static/ui/app/components/CreateDataPack/ExportInfo.js

```diff
@@ -62,6 +73,9 @@ export class ExportInfo extends React.Component {
                 projectName: event.target.value
             });
         }, 250);
+
+        // listen for screensize updates
+        window.addEventListener('resize', this.screenSizeUpdate);
```

Comment:
I dont think we need a listener here anymore since we have the listener in the Application.js file